django/contrib/admin/sites.py

```diff
@@ -259,6 +260,9 @@ def wrapper(*args, **kwargs):
                 return self.admin_view(view, cacheable)(*args, **kwargs)
 
             wrapper.admin_site = self
+            # login_url attribute is used by LoginRequiredMiddleware so it
+            # redirects to admin login.
+            wrapper.login_url = reverse_lazy("admin:login")
```

Comment:
```suggestion
            wrapper.login_url = reverse("admin:login", current_app=self.name)
```
(also wondering if we should reference `login_url` above in the `redirect_to_login` call 🤔

Reply:
We need to lazily reverse because url configuration is not loaded yet. I added the missing argument though.